less/about/preferences.less

```diff
@@ -783,9 +820,60 @@ div.nextPaymentSubmission {
 }
 
 #bitcoinDashboard {
+
+  .modal {
+    .qrcodeOverlay {
+      border: 0;
+      width: 350px;
+      height: auto;
+      margin: 100px auto 0 auto;
+      background: white;
+      box-shadow: @dialogShadow;
+
+      &.fade {
+        transition: @transitionFast;
+      }
+
+      button.close span {
+        color: @gray;
+        font-size: 2.5em;
+      }
+
+      img {
+        clear: both;
+        display: block;
+        margin: 0 auto;
+      }
+
+      .dialog-header {
```

Comment:
Because any modal creates .dialog-header, .dialog-body, .dialog-footer, they should be set inside it, whether they have properties or not. This will keep the style consistency and definitely prevent unexpected regressions.